README.md

```diff
@@ -1,38 +1,131 @@
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+As some [MySejahtera](https://mysejahtera.malaysia.gov.my/intro_en/) user do not receive any notification and missed their appointment, or user ar force to check the application every day for the very same reason.
+<br>
+This application will provide a simple, easier way to get notified on user's vaccination appointment.
+
+### Built With
+
+-   [React JS](https://reactjs.org/)
+-   [Next JS](https://nextjs.org/)
+-   [AWS](https://aws.amazon.com/)
+
+<!-- GETTING STARTED -->
+
 ## Getting Started
 
-First, run the development server:
+### Prerequisites
+
+This is an example of how to list things you need to use the software and how to install them.
+
+-   npm
+-   yarn
+-   node.js v14 
+```sh
+npm install yarn@latest -g
+```
+
+### Installation
+
+1. Clone the repo
 
-```bash
+```sh
+git clone https://github.com/nubpro/nakvaksin.git
+```
+
+2. Install NPM packages
+
+```sh
+yarn install
+```
+
+<!-- USAGE EXAMPLES -->
+
+## Usage
+
+After that, simply run below command to start up the development server
+
+```sh
 yarn dev
 ```
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on
+[http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory agre treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+The `pages/api` directory is map to `/api/*`. Files in this directory is treat as
+[API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
--   [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
--   [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+## Technical Explanation
 
-## How MySejahtera Authentication Works
+#### How MySejahtera Authentication Works
 
 _A. Token Authentication Method_
 
 1. Login to MySejahtera API
+
 2. A token is returned (`x-auth-token` from response header)
+
 3. This token expires within 3 days
+
 4. Pass the token when making MySejahtera API request. In the response header, a new `x-auth-token` is returned.
+
 5. Save that token on the client to ensure token validity.
 
 _B. Login with Basic Auth_
+
 To be documented soon, not something we may need to know?
 
-## Backend Architecture
```

Comment:
Lets get ride of the entire Backend Arch subsection as it is no longer in use (and not followed)